test/Prefixer.js

```diff
@@ -1,9 +1,14 @@
 import {compile, serialize, stringify, middleware, prefixer, prefix} from "../index.js"
 
+const globalCssValues = ['inherit', 'initial', 'unset', 'revert', 'revert-layer']
```

Comment:
https://github.com/thysultan/stylis/pull/289/files#r854961406

Reply:
Can inline it in the single test case.

Reply:
@thysultan  `globalCssValues` is being used twice (That's why I extracted it in the first place). One for `display` and one for `position`. And in the future PR I will add more test cases related with it.